pkg/sql/codegen/pai/template_tf.go

```diff
@@ -164,8 +164,14 @@ if is_estimator:
 else:
     model.load_file("{{.OSSModelDir}}", "model_save")
 
+model_import_name = sqlflow_submitter.get_import_name("""{{.Estimator}}""")
+try:
+    globals()[model_import_name] = __import__(model_import_name)
+except Exception as e:
+    print("failed to import %s: %s" % (model_import_name, e))
```

Comment:
This segment of code appears 5 times in this PR.  It seems should be defined as a function

```python
def sqlflow_submitter.import_model_def(estimator_cls_name):
    model_import_name = sqlflow_submitter.get_import_name(estimator_cls_name)
    try:
        globals()[model_import_name] = __import__(model_import_name)
    except Exception as e:
        print("failed to import %s: %s" % (model_import_name, e))
```

and we call it here as

```python
sqlflow_submitter.import_model_def("""{{.Estimator}}""")
```